### PR TITLE
Rewrote the readme to be inline with issue: Fix readme #27

### PR DESCRIPTION
**Pull request by jwhitt3r**
This PR addresses the issues outline in Fix readme #27 

I have reworked the page to link to the master document, included the contributing section to align with the contents of the document.

* Updated the features to be inline with the webpage
* Added a roadmap linking the master document
* Added a license section linking to your license model, I have removed the MIT license at the top as that is confusing with the actual document
* Added a contact section
* Added an acknowledgement section listing contributors
* Reworded the opening paragraph
* Reworked the page to include the Prerequisites of an OpenAI API key

### CODE REVIEW

**Comment by nang-dev** [comment]
everything else looks good

**Comment by jwhitt3r** [comment]
Glad everything looks good. Let me know if there are any issues ☺️

**Comment by jwhitt3r** [comment]
Yea sure thing! Not a problem at all.

**Comment by jwhitt3r** [comment]
All done.

**Review by gedeondoescode — APPROVED** [review]
Looks good! @nang-dev

File: README.md [hunk 1]
```diff
@@ -1,77 +1,72 @@
...
+  path: '/code/pearai/out/vs/code/electron-main/main.js',
   phase: 'loading',
   moduleId: 'vs/code/electron-main/main',
   neededBy: [ '===anonymous1===' ]
 }
 ```
-- Remove the build and re-ran script `rm -rf out`
-- `./scripts/code.sh`
+To resolve this, follow the below steps:
+ 1. Remove the build and re-ran script `rm -rf out`
```
Comment (Fryingpannn) [inline README.md:52]:
```suggestion
 1. Remove the build: `rm -rf out`
```

File: README.md [hunk 2]
```diff
@@ -1,77 +1,72 @@
...
+ 2. Re run `./scripts/code.sh` and retry
```
Comment (Fryingpannn) [inline README.md:53]:
```suggestion
 2. Re-run the app: `./scripts/code.sh`
```

File: README.md [hunk 3]
```diff
@@ -1,77 +1,72 @@
 # Pear: The Best AI-Powered Code Editor
 
-Supercharge your development in an editor designed for less coding, with AI. This repository is a fork of VSCode.
+Supercharge your development in an editor designed for less coding, with AI. Forked from VSCode, PearAI aims to deliver an enhanced development experience for all.
```
Comment (Fryingpannn) [inline README.md:3]:
```suggestion
Speed up your development with an editor integrated with AI. PearAI is forked from VSCode, and aims to reduce the time from ideation to conception for your product development by achieving the most seamless integration with AI. By developers, for developers.
```

File: README.md [hunk 4]
```diff
@@ -1,77 +1,72 @@
...
 
-This is the main app for PearAI. The bulk of the functionality is within `extension/pearai`. We recommend simply working within that submodule, by cloning https://github.com/trypear/pearai-app/.
+This repository serves as the primary application for PearAI, with most functionalities housed within the extension/pearai submodule. We recommend focusing your work within this submodule by cloning it from https://github.com/trypear/pearai-app/.
```
Comment (Fryingpannn) [inline README.md:5]:
```suggestion
This repository serves as the primary application for PearAI, with most functionalities housed within the `extension/pearai` folder. We recommend focusing your work within this submodule by cloning it from https://github.com/trypear/pearai-app/.
```

**Review by Fryingpannn — CHANGES_REQUESTED** [review]
Nice work. Let's get these changes done then we can merge!

**Comment by nang-dev** [comment]
Github force closed this upon refactor :(

**Comment by jwhitt3r** [comment]
Can you reopen this?

File: README.md (hunk 1 above)
Comment (jwhitt3r) [reply README.md]: I have resolved this.

**Pull request closed without merging**